plugins/coverimage.koplugin/main.lua

```diff
@@ -190,57 +622,24 @@ function CoverImage:addToMainMenu(menu_items)
             -- menu entry: filename dialog
             {
                 text = _("Set image path"),
+                help_text_func = function()
+                    local text = self.cover_image_path
+                    text = text ~= "" and text or _("not set")
+                    return T(_("The actual cover image path is:\n%1"), text)
```

Comment:
```suggestion
                    return T(_("The current cover image path is:\n%1"), text)
```

Reply:
You can just say "current cache path"

Reply:
Oops, wrong line, but the same applies here with cover image.

Reply:
OK